src/shogun/machine/BaggingMachine.cpp

```diff
@@ -44,17 +47,68 @@ CBaggingMachine::~CBaggingMachine()
 
 CBinaryLabels* CBaggingMachine::apply_binary(CFeatures* data)
 {
-	SGVector<float64_t> combined_vector = apply_get_outputs(data);
+	SGMatrix<float64_t> output = apply_outputs_without_combination(data);
+
+	CMeanRule* mean_rule = new CMeanRule();
+
+	SGVector<float64_t> labels = m_combination_rule->combine(output);
+	SGVector<float64_t> probabilities = mean_rule->combine(output);
+
+	CBinaryLabels* pred = new CBinaryLabels(labels);
+  pred->set_values(probabilities);
```

Comment:
make sure your text editor doesnt hide formatting issues from you. In fact, one of the checks we run is a style checker, see its output how to convert the files to the right formatting and how to run locally